Ambit/Source/Ambit/Actors/SpawnerConfigs/SpawnerBaseConfig.cpp

```diff
@@ -164,10 +165,10 @@ void FSpawnerBaseConfig::DeserializeFromJson(
         {
             const FSoftClassPath ClassPath(PathName->AsString());
             const auto& Actor = ClassPath.TryLoadClass<UObject>();
-            if (!Actor)
+            if (Actor == nullptr)
```

Comment:
It is out of scope of this CR but I am thinking whether IsValid() is a better one for UE4 to check the validation of the actor.

Reply:
Agreed, I don't think the clean up is smart enough to figure this one out, unfortunately.